pkg/registry/common/expire/nse_server_test.go

```diff
@@ -34,6 +36,32 @@ import (
 	"github.com/networkservicemesh/sdk/pkg/registry/core/adapters"
 )
 
+type remoteNSEServer struct{}
+
+func (n *remoteNSEServer) Register(ctx context.Context, nse *registry.NetworkServiceEndpoint) (*registry.NetworkServiceEndpoint, error) {
+	return nse.Clone(), nil
+}
+
+func (n *remoteNSEServer) Find(query *registry.NetworkServiceEndpointQuery, s registry.NetworkServiceEndpointRegistry_FindServer) error {
+	return next.NetworkServiceEndpointRegistryServer(s.Context()).Find(query, s)
+}
+
+func (n *remoteNSEServer) Unregister(ctx context.Context, nse *registry.NetworkServiceEndpoint) (*empty.Empty, error) {
+	return next.NetworkServiceEndpointRegistryServer(ctx).Unregister(ctx, nse)
+}
+
+func Test_ExpireServer_ShouldCorrectlySetExpirationTime_InRemoteCase(t *testing.T) {
```

Comment:
Probably it would be better to rework existing tests using `remoteNSEServer`:
```go
c := next.NewNetworkServiceEndpointRegistryClient(
	refresh.NewNetworkServiceEndpointRegistryClient(refresh.WithChainContext(ctx)),
	adapters.NetworkServiceEndpointServerToClient(next.NewNetworkServiceEndpointRegistryServer(
		new(remoteNSEServer), // <-- GRPC invocation
		expire.NewNetworkServiceEndpointRegistryServer(testPeriod*2),
		new(remoteNSEServer), // <-- GRPC invocation
		memory.NewNetworkServiceEndpointRegistryServer(),
	)),
)
```

Reply:
I like this short version. Removed another test. Do you have any concerns?